libs/core/src/lib/panel/panel.component.ts

```diff
@@ -52,42 +47,37 @@ export class PanelComponent implements CssClassBuilder, OnChanges, OnInit, OnDes
     @HostBinding('attr.id')
     id: string = 'fd-panel-' + panelUniqueId++;
 
+    /** Id of the expand button */
+    @Input()
+    expandId: string = 'fd-panel-expand-' + panelExpandUniqueId++;
```

Comment:
Shouldn't this be applied as `[id]` parameter to expand button?

Reply:
Yes, I used it for aria-controls but it's good to have add the id too. Thanks